torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -204,6 +204,10 @@ bool isScalarOp(const Expr* expr) {
 }
 
 bool hasBlockSync(const Expr* expr, const ThreadPredicateMap& pred_map) {
+  if (expr->isA<kir::BlockSync>()) {
+    return true;
+  }
+
```

Comment:
This is the fix for https://github.com/csarofeen/pytorch/issues/1926

Reply:
😮